src/services/handler/__test__/continue-message-handler.test.js

```diff
@@ -3,17 +3,24 @@ import { transferOutFragments } from "../../transfer/transfer-out-fragments";
 import continueMessageHandler from "../continue-message-handler";
 import expect from "expect";
 import { Status } from "../../../models/registration-request";
+import {getNhsNumberByConversationId} from "../../database/registration-request-repository";
+import {NhsNumberNotFoundError} from "../../../errors/errors";

```

Comment:
Incredibly small request for consistency! Totally optional 😄

```suggestion
import { getNhsNumberByConversationId } from "../../database/registration-request-repository";
import { NhsNumberNotFoundError } from "../../../errors/errors";
```